consensus/objs/accusationkey.go

```diff
@@ -0,0 +1,68 @@
+package objs
+
+import (
+	"bytes"
+	"encoding/hex"
+
+	"github.com/alicenet/alicenet/errorz"
+	"github.com/alicenet/alicenet/utils"
+)
+
+// AccusationKey stores DB metadata about an accusation
+type AccusationKey struct {
+	Prefix []byte
+	UUID   []byte
+}
+
+// MarshalBinary takes the AccusationKey object and returns
+// the canonical byte slice
+func (a *AccusationKey) MarshalBinary() ([]byte, error) {
+	if a == nil {
```

Comment:
`nil` pointer calls should be allowed to panic. It'll allow to simplify the return to just `[]byte`

Reply:
Looking at other marshalling code in different structs we can see this pattern all over the place.

Reply:
panic is the way to go 👍 

✅